[INFO -- 2018-11-27 16:44:18]
Starting flip_value script
version: 1.1
[INFO -- 2018-11-27 16:44:18]
Initialization strings:
[INFO -- 2018-11-27 16:44:18]
(1) file /tmp/lreg/lreg
[INFO -- 2018-11-27 16:44:18]
(2) set args /tmp/lreg/randInput /tmp/lreg/output
[INFO -- 2018-11-27 16:44:18]
initSignal:0
[INFO -- 2018-11-27 16:44:18]
endSignal:3
[INFO -- 2018-11-27 16:44:18]
faultModel:2
[EXCEPTION -- 2018-11-27 16:44:20]
pythonException: empty range for randrange() (0,0, 0)
[EXCEPTION -- 2018-11-27 16:44:20]
pythonException: empty range for randrange() (0,0, 0)
[INFO -- 2018-11-27 16:44:20]
Backtrace BEGIN:
#0  __GI_raise (sig=sig@entry=6) at ../sysdeps/unix/sysv/linux/raise.c:51
#1  0x00007ffff6ec6801 in __GI_abort () at abort.c:79
#2  0x00007ffff7ad88b7 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#3  0x00007ffff7adea06 in ?? () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#4  0x00007ffff7adea41 in std::terminate() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#5  0x00007ffff7adec74 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x00007ffff7b06ed2 in std::__throw_bad_alloc() () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#7  0x00005555555582aa in __gnu_cxx::new_allocator<double>::allocate (this=0x7fffffffd930, __n=18446744073709551609) at /usr/include/c++/7/ext/new_allocator.h:102
#8  0x0000555555558043 in std::allocator_traits<std::allocator<double> >::allocate (__a=..., __n=18446744073709551609) at /usr/include/c++/7/bits/alloc_traits.h:436
#9  0x0000555555557b1a in std::_Vector_base<double, std::allocator<double> >::_M_allocate (this=0x7fffffffd930, __n=18446744073709551609) at /usr/include/c++/7/bits/stl_vector.h:172
#10 0x000055555555787f in std::_Vector_base<double, std::allocator<double> >::_M_create_storage (this=0x7fffffffd930, __n=18446744073709551609) at /usr/include/c++/7/bits/stl_vector.h:187
#11 0x0000555555556f4f in std::_Vector_base<double, std::allocator<double> >::_Vector_base (this=0x7fffffffd930, __n=18446744073709551609, __a=...) at /usr/include/c++/7/bits/stl_vector.h:138
#12 0x00005555555569fb in std::vector<double, std::allocator<double> >::vector (this=0x7fffffffd930, __x=std::vector of length -6, capacity 8 = {...}) at /usr/include/c++/7/bits/stl_vector.h:328
#13 0x0000555555556565 in main (argc=3, argv=0x7fffffffde58) at lreg.cpp:259

46	../sysdeps/unix/sysv/linux/raise.c: No such file or directory.

Backtrace END
Fliping a bit of the value pointed by a pointer
Fliping value of field: _M_impl
Fliping value of field: _M_end_of_storage
Fliping a bit of the pointer
Memory content before bitflip:0x000x000x000x000x000x000x000x00
Fault Model: Random bit-flip
base address to flip value: 0x7fffffffd940
address max offset: 8
Memory content after  bitflip:0x860xad0x9f0xa30x550xdc0x210x7d
frame name: std::_Vector_base<double, std::allocator<double> >::_M_allocate
symbol name: this
symbol filename: lreg.cpp
symbol line: 0
value: 0x7fffffffd930
value address: 0x7fffffffd7e8
Type: The type is a pointer.
Type sizeof: 8

[INFO -- 2018-11-27 16:44:20]
Fault Injection Successful after 2.946016311645508s
[INFO -- 2018-11-27 16:44:21]
event type: exit
[EXCEPTION -- 2018-11-27 16:44:21]
exit code: no exit code available
